sync-quickstart-android/build.gradle

```diff
@@ -65,7 +65,9 @@ configurations {
 
 dependencies {
     implementation "androidx.appcompat:appcompat:$androidXVersion"
-    implementation 'com.twilio:sync-android-with-symbols:1.0.0'
+    implementation 'com.twilio:sync-android-with-symbols:1.0.+'
+    // or without symbols:
```

Comment:
We should add the same into the conversation demo app as well

Reply:
Sure, go ahead.